build.py

```diff
@@ -1514,6 +1540,34 @@ def add_cpu_libs_to_linux_dockerfile(backends, target_machine):
     return df
 
 
+def change_default_python_version_rhel(version):
+    df = """
+# The python library version available for install via 'yum install python3.X-devel' does not
+# match the version of python inside the RHEL base container. This means that python packages
+# installed within the container will not be picked up by the python backend stub process pybind
+# bindings. It must instead must be installed via pyenv.
+ENV PYENV_ROOT=/tmp/pyenv_build
```

Comment:
I don't know if it good idea to store it in `/tmp` folder. 
This folder is subject to store short-lived temporary files and may be cleaned up on OS restart.

Reply:
Where would you prefer I store it? Traditionally this is where we've stored our dependencies, such as `boost`.

Additionally, I think it is meant to be short-lived. Recall that we're only interested in gathering the python libs, which we extract with the other binaries we ship. Since this is container-based, I'm not concerned about a restart.

Reply:
`/opt` is a first thing that coming to my mind.